packages/flutter_localizations/lib/src/l10n/cupertino_te.arb

```diff
@@ -22,5 +22,6 @@
   "tabSemanticsLabel": "$tabCountలో $tabIndexవ ట్యాబ్",
   "modalBarrierDismissLabel": "విస్మరించు",
   "searchTextFieldPlaceholderLabel": "సెర్చ్ చేయి",
-  "noSpellCheckReplacementsLabel": "No Replacements Found"
+  "noSpellCheckReplacementsLabel": "No Replacements Found",
+  "lookUpButtonLabel": "Look Up"
```

Comment:
some of these files seem to be localized properly.

Reply:
not entirely sure how to handle the translation. @justinmc probably knows

Reply:
I believe the english is the default and it gets properly localized at a later date
This is what I followed https://github.com/flutter/flutter/tree/master/packages/flutter_localizations#adding-a-new-string-to-localizations

Reply:
Also I added both material and cupertino translations - is that right or do I only need one of them?

Reply:
I think you're right but I always mess this up. @thkim1011 can you confirm this localization string was added correctly?

Reply:
The string looks like it was added properly. One thing I am concerned about is that in the material localizations, "Select all" is cased with a lowercase "a" in "all" so I'm wondering if it should be "Look up" instead with a lowercase "u".

Reply:
I think it's a little different because Look Up is the name of a feature in iOS, and Select all is more of a... description? What do you think?

Reply:
Just saw the native iOS behavior is upper case for both "Select All" and "Look Up". Not sure about android tho.